actions.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ondatra
+
+import (
+	"fmt"
+	"testing"
+
+	"golang.org/x/net/context"
+
+	"github.com/openconfig/ondatra/binding"
+	"github.com/openconfig/ondatra/internal/ate"
+)
+
+// Actions is an ATE Action API.
+type Actions struct {
+	ate binding.ATE
+}
+
+func (action *Actions) String() string {
+	return fmt.Sprintf("{ate: %s}", action.ate)
+}
+
+// BGPPeerNotification is an actions API to send BGP notification/error codes
+type BGPPeerNotification struct {
+	ate                 binding.ATE
+	notificationCode    int
```

Comment:
"notification" in the field names is a bit redundant, and go style doesn't like redundancy. Please rename these fields to "code" and "subCode"